### PR TITLE
Post beta

### PR DESCRIPTION
**Pull request by jhoongo**

### CODE REVIEW

File: FRUI/FRUI/UIElements/DropDownButton/FRActionSheetButton.swift [hunk 1]
```diff
@@ -0,0 +1,145 @@
...
+//
+
+import UIKit
+
+@objc
+public protocol FRActionSheetProtocol {
+    @objc
+    func selectedItem(index: Int, item: String)
+}
+
```
Comment (rodrigoareis) [inline FRUI/FRUI/UIElements/DropDownButton/FRActionSheetButton.swift]: I don't know if it's intentional, but there is no inline documentation for most of FRUI classes. If it is intentional, please ignore this comment.

File: FRAuth/FRAuth/Auth/Node.swift [hunk 2]
```diff
@@ -268,7 +268,7 @@ public class Node: NSObject {
     func buildAuthServiceRequest() -> Request {
         
         var header: [String: String] = [:]
-        header[OpenAM.xRequestedWith] = OpenAM.xmlHTTPRequest
+        header[OpenAM.acceptAPIVersion] = OpenAM.apiResource21 + "," + OpenAM.apiProtocol10
```
Comment (rodrigoareis) [inline FRAuth/FRAuth/Auth/Node.swift]: I would recommend add inline comments regarding this supported APIs in the SDK.

File: FRAuth/FRAuth/Callbacks/MetadataCallback.swift [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+        if let callbackId = json["_id"] as? Int {
+            self._id = callbackId
+        }
+        
+        try super.init(json: [:])
+        self.type = callbackType
+        self.response = json
+    }
+    
+    public override func buildResponse() -> [String : Any] {
```
Comment (rodrigoareis) [inline FRAuth/FRAuth/Callbacks/MetadataCallback.swift]: Missing documentation

File: FRAuth/FRAuth/Auth/AuthService.swift [hunk 4]
```diff
@@ -224,7 +224,7 @@ public class AuthService: NSObject {
     func buildAuthServiceRequest() -> Request {
         
         var header: [String: String] = [:]
-        header[OpenAM.acceptAPIVersion] = OpenAM.apiResource21 + "," + OpenAM.apiProtocol10
```
Comment (rodrigoareis) [inline FRAuth/FRAuth/Auth/AuthService.swift]: I see it was changed from a previous commit. If you still think it is valid, please add comments about supported APIs in the SDK.

File: FRAuth/FRAuth/Auth/Node.swift (hunk 2 above)
Comment (jhoongo) [reply FRAuth/FRAuth/Auth/Node.swift]: Added

File: FRAuth/FRAuth/Callbacks/MetadataCallback.swift (hunk 3 above)
Comment (jhoongo) [reply FRAuth/FRAuth/Callbacks/MetadataCallback.swift]: Added

File: FRAuth/FRAuth/Auth/AuthService.swift (hunk 4 above)
Comment (jhoongo) [reply FRAuth/FRAuth/Auth/AuthService.swift]: Particular change was referenced from old commit; the change has been reverted. Added clarification on API version.

File: FRUI/FRUI/UIElements/DropDownButton/FRActionSheetButton.swift (hunk 1 above)
Comment (jhoongo) [reply FRUI/FRUI/UIElements/DropDownButton/FRActionSheetButton.swift]: Some UI elements in FRUI SDK are missing documentations which is intended. Will add more documentation and proper guideline for UI framework soon.

**Pull request merged**